package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "stylelint-config-web-scrobbler",
-  "version": "0.2.0",
+  "name": "@web-scrobbler/stylelint-config",
```

Comment:
I guess we should update the readme too with this? Or shall we do it after once it's published on NPM?

Reply:
Readme is bundled with the package iirc, so better change it first